homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css

```diff
@@ -0,0 +1,232 @@
+/*header*/
+
+.header {
```

Comment:
Please make the page appearance closer to design view:
- logo top margin
- align of the h1
- font of the text
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/035e4ed9-b35d-4819-ba97-4c4efceebb0f)

Reply:
fixed